config/initializers/redis.rb

```diff
@@ -0,0 +1,8 @@
+env = Figaro.env
+ttl = env.service_provider_request_ttl_hours || DEFAULT_TTL_HOURS
```

Comment:
doesn't the constant `DEFAULT_TTL_HOURS` need to be qualified? Like `ServiceProviderRequestProxy:: DEFAULT_TTL_HOURS`